agent/agent_integration_test.go

```diff
@@ -1,97 +0,0 @@
-package agent
```

Comment:
I removed this because it's covered by TestAbort in agent_test.go

Reply:
covered? or just duplicated?

Reply:
Covered. The test in agent_test.go actually does slightly more than the integration test.